src/components/Header/Header.tsx

```diff
@@ -1,34 +1,43 @@
-import React, { useMemo, useState } from 'react';
-import { Link, useLocation } from 'react-router-dom';
 import { Box, useMediaQuery } from '@material-ui/core';
 import { useTheme } from '@material-ui/core/styles';
-import { useWalletModalToggle } from 'state/application/hooks';
-import {
-  isTransactionRecent,
-  useAllTransactions,
-} from 'state/transactions/hooks';
-import { TransactionDetails } from 'state/transactions/reducer';
-import { shortenAddress, addMaticToMetamask, isSupportedNetwork } from 'utils';
-import useENSName from 'hooks/useENSName';
-import { WalletModal } from 'components';
-import { useActiveWeb3React } from 'hooks';
-import QuickIcon from 'assets/images/quickIcon.svg';
-import QuickLogo from 'assets/images/quickLogo.png';
-import { ReactComponent as ThreeDotIcon } from 'assets/images/ThreeDot.svg';
 import { ReactComponent as LightIcon } from 'assets/images/LightIcon.svg';
-import WalletIcon from 'assets/images/WalletIcon.png';
 import NewTag from 'assets/images/NewTag.png';
+import QuickIcon from 'assets/images/quickIcon.svg';
+import QuickLogo from 'assets/images/quickLogo.png';
+import SparkleBottom from 'assets/images/SparkleBottom.svg';
 import SparkleLeft from 'assets/images/SparkleLeft.svg';
 import SparkleRight from 'assets/images/SparkleRight.svg';
 import SparkleTop from 'assets/images/SparkleTop.svg';
-import SparkleBottom from 'assets/images/SparkleBottom.svg';
+import { ReactComponent as ThreeDotIcon } from 'assets/images/ThreeDot.svg';
+import WalletIcon from 'assets/images/WalletIcon.png';
+import { WalletModal } from 'components';
 import 'components/styles/Header.scss';
+import { useActiveWeb3React } from 'hooks';
+import useENSName from 'hooks/useENSName';
+import React, { useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { Link, useLocation } from 'react-router-dom';
+import { useWalletModalToggle } from 'state/application/hooks';
+import {
+  isTransactionRecent,
+  useAllTransactions,
+} from 'state/transactions/hooks';
+import { TransactionDetails } from 'state/transactions/reducer';
+import { addMaticToMetamask, isSupportedNetwork, shortenAddress } from 'utils';
 
 const newTransactionsFirst = (a: TransactionDetails, b: TransactionDetails) => {
   return b.addedTime - a.addedTime;
 };
 
+const getPredictionPath = () => {
```

Comment:
We don't need this func.  Let's rely on the fact that the env file should be correct and populated

Reply:
ok I will remove it